### PR TITLE
Add consolidated release notes for 2.13.0

### PR DESCRIPTION
**Pull request by gaiksaya**
### Description
Add consolidated release notes for 2.13.0

### Issues Resolved
#4433 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-release-notes-2.13.0.md [hunk 1]
```diff
@@ -0,0 +1,768 @@
...
+<h2>NEW FEATURES</h2>
+
+<h2>EXPERIMENTAL FEATURES</h2>
+
+<h2>RELEASE DETAILS</h2>
+<ul>
+<li>[OpenSearch and OpenSearch Dashboards 2.13.0](https://opensearch.org/versions/opensearch-2-3-0.html) includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance and refactoring updates.
+</li>
+<li>
+OpenSearch <a href="https://github.com/opensearch-project/OpenSearch/blob/2.12/release-notes/opensearch.release-notes-2.13.0.md">Release Notes</a>.
```
Comment (shwetathareja) [inline release-notes/opensearch-release-notes-2.13.0.md:13]: there is typo in link 2.12 to be replaced by 2.13

https://github.com/opensearch-project/OpenSearch/blob/2.13/release-notes/opensearch.release-notes-2.13.0.md

Comment (gaiksaya) [reply release-notes/opensearch-release-notes-2.13.0.md]: Thanks! Fixed it.

**Review by peterzhuamazon — COMMENTED** [review]
Hi @gaiksaya At least these needs to go into infra side of the releases:
* [Defaut config permission too relaxed in deb and rpm packages #3815](https://github.com/opensearch-project/opensearch-build/issues/3815) Mention on releasenotes
* [Debian(RPM later): Restart OpenSearch/Dashboards Service after Upgrading the Package](https://github.com/opensearch-project/opensearch-build/issues/3891) Mention on releasenotes

File: release-notes/opensearch-release-notes-2.13.0.md [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+<h3>OpenSearch Security Analytics Dashboards</h3>
+
+<ul>
+<li>Updated cypress version to match core OSD (<a href="https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/907">#907</a>)</li>
+</ul>
+
+<h3> OpenSearch Build </h3>
+
+<ul>
+<li>Fix default config permissions for RPM and Debian Packages (<a href="https://github.com/opensearch-project/opensearch-build/issues/3815">#3815</a>)</li>
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-2.13.0.md:505]: `Tighten default permissions associated with access to installed files for RPM and Debian Packages`
Use these two PR (https://github.com/opensearch-project/opensearch-build/pull/4043) (https://github.com/opensearch-project/opensearch-build/pull/4546)

File: release-notes/opensearch-release-notes-2.13.0.md [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+<li>Fix OpenSearch and OpenSearch-Dashboards Service restart after upgrading the package (<a href="https://github.com/opensearch-project/opensearch-build/issues/3891">#3891</a>)</li>
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-2.13.0.md:506]: `Allow automatically restarting of OpenSearch and OpenSearch-Dashboards Service after upgrading the Debian package`

Use this PR: https://github.com/opensearch-project/opensearch-build/pull/4530

**Review by bbarani — APPROVED** [review]

**Pull request merged**